pure.zsh

```diff
@@ -563,6 +558,11 @@ prompt_pure_async_callback() {
 					;;
 			esac
 			;;
+		prompt_pure_async_git_stash)
+			local prev_stash=$prompt_pure_git_stash
+			prompt_pure_git_stash=$output
```

Comment:
We need to explicitly define this as a global so as to not trigger error messages when `setopt WARN_CREATE_GLOBAL` is used.

```suggestion
			typeset -g prompt_pure_git_stash=$output
```